_docs_operate/connector-cli-operations.md

```diff
@@ -0,0 +1,65 @@
+---
+# Start automatic generation
+permalink: operate/connector-cli-operations
+published: true
+title: "Connector CLI Operations"
+type: scenario
+toc: true
+properties:
+  - id: SC124
+  - category: Connector Operations
+  - description:
+  - customer:
+  - component: operate
+  - level:
+  - implementation status: DOCS ONLY
+  - documentation status: DONE
+  - published: true
+  - link: connector-cli-operations
+require:
+required_by:
+# End automatic generation
+---
+
+The following guide provides step-by-step instructions for [configuring](#setup) the command-line interface (CLI) of the Connector and performing essential [operations](#operations) using it.
+
+## Setup
+
+Different Connector setups may require different approaches to executing the available commands provided by the Connector CLI.
+
+### Setup with Docker Compose
```

Comment:
all of this is a bit to explanatory. maybe give an example on how to switch the command